transducers.js

```diff
@@ -4,7 +4,7 @@
 var symbolExists = typeof Symbol !== 'undefined';
 
 var protocols = {
-  iterator: symbolExists ? Symbol.iterator : '@@iterator'
+  iterator: symbolExists && Symbol.iterator || '@@iterator'
```

Comment:
This is my only complain: can you revert this? I think it's less readable. Everything else looks good!